File: algorithms/hackerrank/camelcase/camelcase.py
#!/usr/bin/env python3
# https://www.hackerrank.com/challenges/camelcase

import unittest


def camelcase(s):
    count = 1
    for character in s:
        if character.isupper():
            count += 1
    return count


class TestCode(unittest.TestCase):
    def test_example(self):
        self.assertEqual(5, camelcase('saveChangesInTheEditor'))

File: algorithms/code/hackerrank/camelcase/camelcase.go
// Package camelcase implements https://www.hackerrank.com/challenges/camelcase
package camelcase

import (
	"unicode"
)

// Camelcase - implements the solution to the problem
func Camelcase(s string) int32 {
	var count int32 = 1
	for _, character := range s {
		if unicode.IsUpper(character) {
			count++
		}
	}
	return count
}
